### PR TITLE
Redis docker

### PR DESCRIPTION
**Pull request by SirMorfield**
No more upstash, we host it ourselves

### CODE REVIEW

**Review by LithiumOx — APPROVED** [review]
For the record Upstash is still nice, the person pushing 1.9 million pixels without seeing any changes on the canvas should get community service.

Apart from my little rant, code look fine just fine!

File: docker-compose.yaml
Comment (Obult) [inline docker-compose.yaml:1]: Also possible to use a named volume (not mounted) for the redis DB container if we do not need access to the folder directly.

Comment (Obult) [reply docker-compose.yaml:1]: example:

redis:
    volumes:
        - redisDB:/data
        {etc}
        
volumes:
    redisDB:

**Review by Obult — APPROVED** [review]
small change suggestion for docker volumes with DB

**Pull request merged**